### ISSUE TITLE
Re-design highlighting foundation

### ISSUE THREAD

**Issue by daniellecorke** [ONSdigital/design-system#3109]
### Background

Following #3031 being completed, further guidance has been shared: 

> So I've been given text from WAVE saying:
'Underlines almost universally indicates linked text. Consider removing the underline from the non-link text. Other styling (e.g., bold or italics) can be used to differentiate the text.'

We completed #3031 to solve an issue following the eQ DAC assessment. They are expecting to re-test in June therefore we may need to prioritise this ticket ahead of their next assessment. 

### Tasks
Re-design the highlight text with underline, taking into account DAC feedback in the previous ticket. 

### Acceptance Criteria
An updated version of the highlight text without an underline is available on the Design System.

**Comment by daniellecorke** [comment]
On the UCD Board: https://jira.ons.gov.uk/browse/CU-89.

**Comment by e-livingstone** [comment]
In progress for user research. EQ team are also consulting with DAC re colour contrast and underline.

**Comment by MagdalenaLarge** [comment]
Update from Richard Parker (UR):  Have contacted DAC and received response [Email from DAC regarding CU-89.docx](https://jira.ons.gov.uk/secure/attachment/119358/119358_Email+from+DAC+regarding+CU-89.docx)

They have stated that the current solution on the Design system under 'Highlighting' is sufficient from a WCAG standpoint - https://service-manual.ons.gov.uk/design-system/foundations/typography 

A potential usability concern was raised around users thinking the underlined text is a link. However, the context in which the highlighting is used for surveys means that there is no need for the user to select the wording. 

As our current solution meets WCAG guidelines, there is no further action needed on this. If there is user feedback or analytics showing the underlining is causing a failure for users to complete their tasks in future, this can then be addressed.
